buildtest/cli/__init__.py

```diff
@@ -1140,6 +1140,14 @@ def report_menu(subparsers, parent_parser):
         "--format",
         help="format field for printing purposes. For more details see --helpformat for list of available fields. Fields must be separated by comma (usage: --format <field1>,<field2>,...)",
     )
+
+    filter_group.add_argument(
+        "-a",
+        "--all",
+        action="store_true",
+        help="print all filter fields for --format name,id,user,state,returncode,runtime,outfile,errfile,buildspec options to filter the report",
```

Comment:
these are called format fields so please change `filter` -> `format`. Now that i think about it `--all` seems bit misleading since its not all the format fields but just a pre-selected subset of format fields. What do you think.

Perhaps `--detailed` or `--default` may be more suitable. Any other suggestion. I think it okay to omit out `name,id,user,state,returncode,runtime,outfile,errfile,buildspec` from the help message. This can get out of date from code if we decide to change and is not important for user.

Reply:
**"_these** are called format fields so please change filter -> format_": **Will do!**

_Now that i think about it --all seems bit misleading since its not all the format fields but just a pre-selected subset of format fields. What do you think" I agree with you I am working on the implementation in report.py and using_ "
****--all" for some features in display_table and display_fields very annoying.**** 

"Perhaps --detailed or --default may be more suitable. Any other suggestion." 
****--default to me very close to --all, while --detailed sounds better. What about --main, which we can specify as main features for any report. Just a suggestion:).****  

_"I think it okay to omit out name,id,user,state,returncode,runtime,outfile,errfile,buildspec from the help message. This can get out of date from code if we decide to change and is not important for user"_
**okay, I will do that**